### PR TITLE
Update wasmtime agenda for next meeting

### PR DESCRIPTION
**Pull request by bnjbvr**

### CODE REVIEW

**Review by cfallin — APPROVED** [review]
Thanks!

File: meetings/wasmtime/2021/wasmtime-12-09.md [hunk 1]
```diff
@@ -9,9 +9,13 @@
     1. Please help take notes.
     1. Thanks!
 1. Announcements
-    1. _Sumbit a PR to add your announcement here_
+    1. _Submit a PR to add your announcement here_
 1. Other agenda items
-    1. _Sumbit a PR to add your item here_
+    1. Profiling (with vtune on linux and windows, other approaches on Windows)
```
[the remark this answers is not in the document]
Comment (bnjbvr) [reply meetings/wasmtime/2021/wasmtime-12-09.md:14]: No, really meant other approaches on Windows (e.g. integration with other profilers), but MacOS would be interesting to discuss, so will amend. Thanks!

[the remark this answers is not in the document]
Comment (bnjbvr) [reply meetings/wasmtime/2021/wasmtime-12-09.md:14]:
```suggestion
    1. Profiling (with vtune on Linux and Windows, other profilers on Windows, profiling on MacOS)
```

[the remark this answers is not in the document]
Comment (tschneidereit) [reply meetings/wasmtime/2021/wasmtime-12-09.md]: ah, I misunderstood! Thank you for the clarification!)

**Pull request merged**